[Bug]: [Nightly]Load partitions failed reporting 'index not exist'

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:0a83cdf
- Deployment mode(standalone or cluster):cluster
- SDK version(e.g. pymilvus v2.0.0rc2):2.2.0.dev45
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```


### Current Behavior

Load partitions failed reporting index not exist.
```
[2022-10-17T21:48:33.913Z] [2022-10-17 21:04:34 - ERROR - pymilvus.decorators]: RPC error: [load_partitions], <MilvusException: (code=1, message=index not exist)>, <Time:{'RPC start': '2022-10-17 21:04:34.268512', 'RPC error': '2022-10-17 21:04:34.272631'}> (decorators.py:112)
[2022-10-17T21:48:33.913Z] [2022-10-17 21:04:34 - ERROR - ci_test]: Traceback (most recent call last):
[2022-10-17T21:48:33.913Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 26, in inner_wrapper
[2022-10-17T21:48:33.913Z]     res = func(*args, **_kwargs)
[2022-10-17T21:48:33.913Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 57, in api_request
[2022-10-17T21:48:33.913Z]     return func(*arg, **kwargs)
[2022-10-17T21:48:33.913Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/orm/collection.py", line 471, in load
[2022-10-17T21:48:33.913Z]     conn.load_partitions(self._name, partition_names, replica_number=replica_number, timeout=timeout, **kwargs)
[2022-10-17T21:48:33.913Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 113, in handler
[2022-10-17T21:48:33.913Z]     raise e
[2022-10-17T21:48:33.913Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 109, in handler
[2022-10-17T21:48:33.913Z]     return func(*args, **kwargs)
[2022-10-17T21:48:33.913Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 139, in handler
[2022-10-17T21:48:33.913Z]     ret = func(self, *args, **kwargs)
[2022-10-17T21:48:33.913Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 89, in handler
[2022-10-17T21:48:33.913Z]     raise e
[2022-10-17T21:48:33.913Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 51, in handler
[2022-10-17T21:48:33.913Z]     return func(self, *args, **kwargs)
[2022-10-17T21:48:33.913Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/client/grpc_handler.py", line 720, in load_partitions
[2022-10-17T21:48:33.913Z]     raise MilvusException(response.error_code, response.reason)
[2022-10-17T21:48:33.913Z] pymilvus.exceptions.MilvusException: <MilvusException: (code=1, message=index not exist)>
[2022-10-17T21:48:33.913Z]  (api_request.py:39)
[2022-10-17T21:48:33.913Z] [2022-10-17 21:04:34 - ERROR - ci_test]: (api_response) : <MilvusException: (code=1, message=index not exist)> (api_request.py:40)
```

### Expected Behavior

PASSED

### Steps To Reproduce


https://jenkins.milvus.io:18080/blue/organizations/jenkins/Milvus%20Nightly%20CI/detail/master/164/pipeline/187



### Milvus Log

1. milvus log: [artifacts-milvus-distributed-pulsar-nightly-164-pymilvus-e2e-logs.tar.gz](https://jenkins.milvus.io:18080/job/Milvus%20Nightly%20CI/job/master/164/artifact/artifacts-milvus-distributed-pulsar-nightly-164-pymilvus-e2e-logs.tar.gz)
2. collection name: coll__OD6MjQOs
3. failed time: [2022-10-17T21:04:36.934Z] [gw2] [ 96%] FAILED testcases/test_utility.py::TestUtilityBase::test_loading_progress_multi_replicas

### Anything else?

_No response_

--- a/tests/python_client/testcases/test_utility.py
+++ b/tests/python_client/testcases/test_utility.py
@@ -908,6 +908,8 @@ class TestUtilityBase(TestcaseBase):
         collection_w = self.init_collection_wrap()
         collection_w.insert(cf.gen_default_dataframe_data())
         assert collection_w.num_entities == ct.default_nb
+        default_index = {"index_type": "IVF_FLAT", "params": {"nlist": 128}, "metric_type": "L2"}
+        collection_w.create_index("float_vector", default_index)
         collection_w.load(partition_names=[ct.default_partition_name], replica_number=2)
         res_collection, _ = self.utility_wrap.loading_progress(collection_w.name)
         assert res_collection == {loading_progress: '100%'}

